configuration.md

```diff
@@ -12,12 +12,12 @@ description: >-
 2. On the **Database Configuration** wizard page specify the **SQL Server**, **Database Name** and authentication. It is possible to overwrite the existing database under the same name.
 3. if you are using the default instance type “servername”, or “servername.yourdomain.loc”
 4. if you are using a named instance type “servername\instancename”, or “servername. Yourdomain.loc\instancenameˇ 
-5. if your SQL Server is on a non-standard port \(different from 1433\), type “servername,port” or “servername\instance,port”. \(FQDN formats are also supported\)  
+5. if your SQL Server is on a non-standard port \(different from 1433\), type “servername.port” or “servername\instance.port”. \(FQDN formats are also supported\)  
```

Comment:
čini se da je ipak ispravno sa zarezom
https://stackoverflow.com/questions/5294721/how-to-specify-a-port-number-in-sql-server-connection-string